src/pages/split-condition/SplitCondition.tsx

```diff
@@ -141,18 +176,15 @@ export const SplitCondition = ({
         />
       )}
 
-      <label htmlFor="amount">Amount</label>
-      <Controller
-        name="amount"
-        rules={{ required: true, validate: (amount) => amount.gt(ZERO_BN) }}
-        control={control}
-        decimals={collateralToken.decimals}
-        as={BigNumberInputWrapper}
-      />
-
-      <button type="submit" disabled={!canSubmit}>
+      <InputAmount
+        collateral={collateralToken}
+        positionId={positionId}
+        splitFrom={splitFrom}
+        formMethods={formMethods}
+      ></InputAmount>
+      <button onClick={handleSubmit(onSubmit)} disabled={!canSubmit}>
         Split
```

Comment:
If  the transaction is running, we could change the text to `Splitting...` or something like that, so the user knows what is happening.